nrf_modem/include/nrf_socket.h

```diff
@@ -17,33 +17,26 @@
 extern "C" {
 #endif
 
-#if !defined(__GNUC__) || (__GNUC__ == 0)
 typedef int32_t ssize_t;
-#else
-#include <sys/types.h>
-#ifdef __SES_ARM
-typedef int32_t ssize_t;
-#endif
-#endif
 
 /**@addtogroup nrf_socket_address_resolution
  *@{
  */
 
-/**@brief Host to network byte-orders on half word. */
-#define NRF_HTONS(val) ((uint16_t)((((val)&0xff00) >> 8) | ((((val)&0x00ff) << 8))))
+/** @brief Host to network byte-orders on half word. */
```

Comment:
```suggestion
/** @brief Host to network byte-orders on half word */
```

Reply:
Either remove punctuation on @brief throughout this file or add punctuation where it is missing.